recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
Personally, I'm not a huge fan. I can see value in pulling all of the variables into one place, but we aren't talking about complex documents in the first place.

The main advantage of pulling the version out, vs anything else, is that it makes it trivial to find out what version the recipe is building. Nobody needs to do that for the md5 and the package name is already at the top of the file.